.github/labeler.yml

```diff
@@ -0,0 +1,36 @@
+build:
+- default.nix
+- CMakeLists.txt
+- build/*
+- build/**/*
+- cmake/*
+- cmake/**/*
+
+code quality:
+- src/test/*
+- src/test/**/*
+- .clang-format
+- .codespell
+- .eslint*
+- .flake8
+- .pre-commit-config.yaml
+- .travis.yml
```

Comment:
Not sure if we should include the CI config files here?

Reply:
ok, I can remove them.